timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -197,3 +197,10 @@ services:
         returns: LeaderTimes
         docs: |
           Version of ConjureTimelockService#leaderTime endpoint for acquiring leaderTimes for a set of namespaces.
+      startTransactions:
```

Comment:
[Possibly non-actionable]: you went from `leaderTime` -> `leaderTimes` when batching, but we already had `startTransactions` as it is batched in some capacity already. No idea if or how you could differentiate (or if it's even worth doing!), but thought I'd call it out anyway.